PROTOCOL.md

```diff
@@ -2334,17 +2411,65 @@ content-type: application/x-ndjson; charset=utf-8
 ### Timestamp Format
 Accepted timestamp format by a delta sharing server: in the ISO8601 format, in the UTC timezone, such as `2022-01-01T00:00:00Z`.   
 
-## API Response Format
+## Delta Sharing Capabilities Header
+This section explains the details of delta sharing capabilities header, which was introduced to help 
+delta sharing catch up with features in [delta protocol](https://github.com/delta-io/delta/blob/master/PROTOCOL.md).
+
+The key of the header is **delta-sharing-capabilities**, the value is semicolon separated capabilities. 
+Each capability is in the format of "key=value1,value2", values are separated by commas.
+Example: "responseformat=delta;readerfeatures=deletionvectors,columnmapping". All keys and values should
+be case-insensitive when processed by the server.
+
+This header can be used in the request for [Query Table Metadata](#query-table-metadata), 
+[Query Table](#read-data-from-a-table), and [Query Table Changes](#read-change-data-feed-from-a-table).
+
+**Compatibility**
+
+<table>
+<tr>
+<th>Client/Server</th>
+<th>Server that doesn't know the header</th>
```

Comment:
"Server that recognizes the header"

Similar below.

Reply:
updated.